### PR TITLE
cleanup: Optimization logging format for pkg/kubelet

### PR DESCRIPTION
**Pull request by sanwishe**
Signed-off-by: sanwishe <jiang.mingzhi35@zte.com.cn>

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:

Optimization structured logging format for pkg/kubelet according to:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
"NONE"
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
"NONE"
```

### CODE REVIEW

File: pkg/kubelet/status/status_manager.go [hunk 1]
```diff
@@ -566,7 +566,7 @@ func (m *manager) syncPod(uid types.UID, status versionedPodStatus) {
 		klog.InfoS("Failed to get status for pod",
 			"podUID", uid,
 			"pod", klog.KRef(status.podNamespace, status.podName),
-			"error", err)
+			"err", err)
```
Comment (wangyysde) [inline pkg/kubelet/status/status_manager.go:569]: Could you tell me why you change "error" to "err" in this line? Thanks. This string will output to user direct. "error" is more friendly to user,isn't it?

Comment (sanwishe) [reply pkg/kubelet/status/status_manager.go:569]: @wangyysde according to the guide:
[name-arguments](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments)

>Examples of good keys (strongly suggested, will be extended when pattern emerge, no standard schema yet):
>
>cacheKey
>cacheValue
>CIDR
> ...
>dockerVersion
>duration
>err
>job

Comment (yangjunmyfm192085) [reply pkg/kubelet/status/status_manager.go:569]: Do use err rather than error to match the key used by klog.ErrorS

**Review by lunhuijie — APPROVED** [review]

File: pkg/kubelet/status/status_manager.go (hunk 1 above)
Comment (ehashman) [reply pkg/kubelet/status/status_manager.go:569]: It's correct to use `err` per the guide.

**Review by ehashman — COMMENTED** [review]
/priority backlog
/triage accepted
/lgtm

**Review by mrunalp — APPROVED** [review]

**Pull request merged**